CHANGELOG.rst

```diff
@@ -17,6 +17,7 @@ Changed
 =======
 
 - Logged admin can see any project (#262)
+- Simpler and safer authentication logic
```

Comment:
nit: When adding a line in the changelog, if possible specify the pull-request number.